src/main/java/dukeclasses/Ui.java

```diff
@@ -0,0 +1,123 @@
+package dukeclasses;
+
+import java.time.DateTimeException;
+
+/**
+ * Handles all the user inputs.
+ */
+public class Ui {
+
+    protected boolean hasEnded;
+    protected TaskManager manager;
+    protected String horizontalLine = "*****************************************************";
+
+    public Ui(TaskManager manager) {
+        this.hasEnded = false;
+        this.manager = manager;
+    }
+
+    /**
+     * Prints out the introduction text.
+     */
+    public void introduction() {
+
+        System.out.println(horizontalLine);
+        //To run using gradle go and search in Help > Delegate Run > build tools > gradle
+        String logo = " _____        .      _     |\n"
+                +     "|         |  | |"
+                +                     "   _| |_   |\n"
+                +      "|_____   _|  | |  |_   _|  |__\n"
+                +      "|_____  |_|  |_|    |_|    |  |";
+
+        System.out.println("" + logo);
+
+        try {
+            Storage store = new Storage();
+            store.checkFileDir();
+            store.checkFile();
+        } catch (Exception ex) {
+            System.out.println(ex);
+        }
+
+        System.out.println("Hello!! My name is Edith. Tony Stark's Personal Assistant" + "\n"
+                + "What can I do for you?");
+        System.out.println(horizontalLine);
+
+    }
+
+    /**
+     * Handles all the different kinds of inputs the user can have.
+     *
+     * @param textEntered textEntered represents whatever the user types
+     */
+    public void handleInputs(String textEntered) {
+
+        System.out.println(horizontalLine);
+
+        if (textEntered.equals("list")) {
+            manager.listAllTasks();
+        } else if (textEntered.contains("done")) {
+            Parser parser = new Parser();
+            int indexOfTaskDone = parser.handleDoneCommands(textEntered);
+            try {
+                manager.setTaskAsDone(indexOfTaskDone);
+            } catch (ArrayIndexOutOfBoundsException ex) {
+                System.out.println("Done must be followed by a number");
+            } catch (IndexOutOfBoundsException ex) {
+                System.out.println("There is no task with that index! ");
+            }
+        } else if (textEntered.contains("todo") || (textEntered.contains("deadline"))
+                || textEntered.contains("event")) { //create a task
+            try {
```

Comment:
You might wanna try converting this entire if-else statement block into a switch case block to make it more succinct!